velodyne_pointcloud/launch/VLP16_points.launch

```diff
@@ -10,13 +10,14 @@
   <arg name="manager" default="$(arg frame_id)_nodelet_manager" />
   <arg name="max_range" default="130.0" />
   <arg name="min_range" default="0.4" />
-  <arg name="pcap" default="" />
+  <arg name="pcap" default="/home/andre/Documents/bcip/bcip_ws/mapping_module/ghetto_board/2018_12_20_car_test/dual_return/2018-12-20-11-59-21-eth0.pcap" />
```

Comment:
I assume this is a relic of testing. Please remove before giving a "final" stamp to this PR.